index.html

```diff
@@ -40,21 +41,186 @@ <h3>I'm a Software Developer.</h3>
+                        <h2>Profesional Art Printing Data</h2>
+                        <p>A daily selection of privately personalized reads
+                             no accounts or sign-ups required. has been the industry's standard
+                        </p>
+                        <div class="languages">
+                            
+                            <a href="#" class="project-language"> html</a>
+                            <a href="#" class="project-language">bootstrap</a>
+                            <a href="#" class="project-language"> ruby</a>
+                        </div>
+                        <a href="#" class="project-btn">See project</a>
+                        
+                     </div> 
+                     <div class="project">
+                        <h2>Profesional Art Printing Data</h2>
+                        <p>A daily selection of privately personalized reads
+                             no accounts or sign-ups required. has been the industry's standard
+                        </p>
+                        <div class="languages">
+                            
+                            <a href="#" class="project-language"> html</a>
+                            <a href="#" class="project-language">bootstrap</a>
+                            <a href="#" class="project-language"> ruby</a>
+                        </div>
+                        <a href="#" class="project-btn">See project</a>
+                        
+                     </div>
+                      <div class="project">
+                        <h2>Profesional Art Printing Data</h2>
+                        <p>A daily selection of privately personalized reads
+                             no accounts or sign-ups required. has been the industry's standard
+                        </p>
+                        <div class="languages">
+                            
+                            <a href="#" class="project-language"> html</a>
+                            <a href="#" class="project-language">bootstrap</a>
+                            <a href="#" class="project-language"> ruby</a>
+                        </div>
+                        <a href="#" class="project-btn">See project</a>
+                        
+                     </div> 
+                     <div class="project">
+                        <h2>Profesional Art Printing Data</h2>
+                        <p>A daily selection of privately personalized reads
+                             no accounts or sign-ups required. has been the industry's standard
+                        </p>
+                        <div class="languages">
+                            
+                            <a href="#" class="project-language"> html</a>
+                            <a href="#" class="project-language">bootstrap</a>
+                            <a href="#" class="project-language"> ruby</a>
+                        </div>
```

Comment:
- [ ] Let us wrap our technologies inside the `ul li tag` as well as increase the size of it to look the same we have on the figma.
The list of project technologies (e.g., "JavaScript", "HTML" and "CSS") are a list ul and each of them a li